src/api/userDetails.js

```diff
@@ -32,3 +33,23 @@ Zeeguu_API.prototype.saveUserDetails = function (
       setErrorMessage("Invalid credentials");
     });
 };
+
+Zeeguu_API.prototype.modifyCEFRlevel = function (
+  languageID,
+  cefrLevel,
+  onSuccess,
+  onError
+) {
+  let payload = {
+    language_id: languageID,
+    //language_reading: languageID,
```

Comment:
why these comments?

Reply:
Legacy comments from the bug-hunt. I removed them now.